scripts/actor.gd

```diff
@@ -32,23 +31,25 @@ signal actor_killed(me: Actor)
 
 func _aim_at(target_position: Vector3):
 	aimpoint = target_position
-	
+
 func _move_direction(input_dir: Vector2):
 	movement_direction = (transform.basis * Vector3(input_dir.x, 0, input_dir.y)).normalized()
 
 func _aim_weapon():
-	# side to side rotation
-	rotator.look_at(controller.get_aim_position(), Vector3.UP)
-	
-	#up and down rotation
-	weapon_base.look_at(controller.get_aim_position(), Vector3.UP)
+	var aim_position = controller.get_aim_position()
+	if !aim_position.is_equal_approx(Vector3.UP):
```

Comment:
Is that for fixing the `Node origin and target are in the same position, look_at() failed.` error?

Reply:
Correct. The look_at function will throw a warning/error if the target vector is approximately up.
```
E 0:00:00:0674   actor.gd:42 @ _aim_weapon(): Up vector and direction between node origin and target are aligned, look_at() failed.
  <C++ Error>    Condition "p_up.cross(p_target - p_pos).is_zero_approx()" is true.
  <C++ Source>   scene/3d/node_3d.cpp:821 @ look_at_from_position()
  <Stack Trace>  actor.gd:42 @ _aim_weapon()
                 actor.gd:76 @ _process()
```